crates/usvg/src/parser/svgtree/parse.rs

```diff
@@ -321,19 +326,23 @@ pub(crate) fn parse_svg_element<'input>(
     };
 
     // Apply CSS.
-    for rule in &style_sheet.rules {
-        if rule.selector.matches(&XmlNode(xml_node)) {
-            for declaration in &rule.declarations {
-                write_declaration(declaration);
-            }
-        }
-    }
+    let mut declarations = style_sheet
```

Comment:
The code `resolve_css` is not enough?

Reply:
That was an accidental diff, I already reverted it.